JWBestPracticeApps/FeedTableViewController/FeedTableViewController/FeedItem/PlayerItemCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  FeedItemCell.swift
+//  FeedTableViewController
+//
+//  Created by Amitai Blickstein on 6/26/22.
+//
+
+import UIKit
+import JWPlayerKit
+
+class PlayerItemCell: UITableViewCell {
+    static let reuseIdentifier = "PlayerItemCell"
+    static let nibName         = "PlayerItemCell"
+            
+    // Provided by viewModel.
+    var item: JWPlayerItem? {
+        didSet { configureCellPlayer(with: item) }
+    }
+    
+    // The view (UIView subclass) underlying this View (in the MVVM sense)
+    @IBOutlet weak var playerView: JWPlayerView!
+    // To show the video #, that the feed is "infinite"
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    
+    private func configureCellPlayer(with item: JWPlayerItem?) {
+        guard let itemConfig = getPlayerConfig(for: item)
+        else { return }
+
+        titleLabel.text = item?.title
+        playerView.player.configurePlayer(with: itemConfig)
```

Comment:
As much as I like this, I wonder if it is more efficient to load an empty default config on the player view when it is first created using `init`. Then, when this configure method is called, call `JWPlayer.loadPlaylist([item])`

My reasoning is that calling `configurePlayer` wipes out the entire player, and reloads everything including the Headless Player's javascript files. All that can be skipped by calling `loadPlaylist`.

Reply:
@s-seibert I incorporated this, but now the cells do not seem to call `loadPlaylist` until after they scroll off/back on screen. Insights?